src/AutoBuild.jl

```diff
@@ -1345,7 +1350,7 @@ const uuid_package = UUID("cfb74b52-ec16-5bb7-a574-95d9e393895e")
 # For even more interesting historical reasons, we append an extra
 # "_jll" to the name of the new package before computing its UUID.
 jll_uuid(name) = bb_specific_uuid5(uuid_package, "$(name)_jll")
-function build_project_dict(name, version, dependencies::Array{Dependency}, julia_compat::String=DEFAULT_JULIA_VERSION_SPEC)
+function build_project_dict(name, version, dependencies::Array{Dependency}, julia_compat::String=DEFAULT_JULIA_VERSION_SPEC, lazy_artifacts::Bool=false)
```

Comment:
I think the callers of `build_project_dict()` will need to pass `lazy_artifacts` through as well.

Reply:
Ooops, good point!  I _think_ I've fixed that